Makefile

```diff
@@ -1316,6 +1317,60 @@ GENERATOR_AOTCPP_TESTS := $(filter-out generator_aotcpp_gpu_multi_context_thread
 
 test_aotcpp_generator: $(GENERATOR_AOTCPP_TESTS)
 
+# Tests below probably don't make much sense for Xtensa.
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_alias,$(GENERATOR_AOTXTENSA_TESTS))
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_async_parallel,$(GENERATOR_AOTXTENSA_TESTS))
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_autograd,$(GENERATOR_AOTXTENSA_TESTS))
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_cxx_mangling,$(GENERATOR_AOTXTENSA_TESTS))
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_cxx_mangling_define_extern,$(GENERATOR_AOTXTENSA_TESTS))
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_metadata_tester,$(GENERATOR_AOTXTENSA_TESTS))
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_string_param,$(GENERATOR_AOTXTENSA_TESTS))
+
+# Tests below work, but need to disable parallel in the schedule.
+# TODO(vksnk): figure out what's wrong with parallel in this case.
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_example,$(GENERATOR_AOTXTENSA_TESTS))
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_mandelbrot,$(GENERATOR_AOTXTENSA_TESTS))
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_pyramid,$(GENERATOR_AOTXTENSA_TESTS))
+
+# Xtensa doesn't have float64 vectors
+GENERATOR_AOTXTENSA_TESTS := $(filter-out generator_aotcpp_xtensa_templated,$(GENERATOR_AOTXTENSA_TESTS))
+
+# Needs define_extent
```

Comment:
For at least some of these, we should probably create Issue(s) and tag them with comment, so there is some tracking of the eventual fixes

Reply:
Done.